app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container-signup">
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] As requested by the previous reviewer, we are supposed to modify and style the registration and login forms to match the requested design. I think you have just aligned the form center without modifying placeholder texts, font weight, font size, input fields and layout. Great design improve our user's experience :+1:  **Your registration and login screenshots**
![Screenshot from 2023-11-23 11-07-56](https://github.com/Abdulwhbhussain/Budget-app/assets/63087491/a7e31452-c264-4e5b-a94f-cb838de42688)
![Screenshot from 2023-11-23 11-24-13](https://github.com/Abdulwhbhussain/Budget-app/assets/63087491/78b14781-6217-412c-b227-8ef2f741b19e)

Reply:
![loign-ss](https://github.com/Abdulwhbhussain/Budget-app/assets/124990680/70539a16-d6b3-4b71-a6c6-44931daa1587)
![Signup-ss](https://github.com/Abdulwhbhussain/Budget-app/assets/124990680/1a934351-a267-49b4-afb9-4a8994c8a15f)
Brother! I did it to the Best of My Ability...